yaqd_wright/_wright_filter_wheel.py

```diff
@@ -1,21 +1,31 @@
 __all__ = ["WrightFilterWheel"]
 
 import asyncio
+from typing import Dict
 import time
+
 from yaqd_core import ContinuousHardware, DiscreteHardware, aserial
+from ._serial import SerialDispatcher
 
 class WrightFilterWheel(ContinuousHardware, DiscreteHardware):
     _kind = "wright-filter-wheel"
+    serial_dispatchers: Dict[str, SerialDispatcher] = {}
 
     def __init__(self, name, config, config_filepath):
         super().__init__(name, config, config_filepath)
         self._motornum=config["motor"]
-        self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
         self._units=config["units"]
         self._microstep=config["microstep"]
         self._set_microstep(self._microstep)
         time.sleep(0.2)
         self._steps_per_rotation=400
+        if config["serial_port"] in WrightFilterWheel.serial_dispatchers:
+            self._serial_port = WrightFilterWheel.serial_dispatchers[config["serial_port"]]
+        else:
+            self._serial_port = SerialDispatcher(
```

Comment:
Just directly set to ASerial object, SerialDispatcher objects are not in this repo, they are for communication protocols that allow you to route messages correctly based only on the message, this device has no such ability